database/read_and_save.py

```diff
@@ -0,0 +1,93 @@
+"""Example of reading and saving some inverter registers."""
+
+import sys
+from datetime import datetime, timedelta
+from os import environ
+
+from solaxx3.solaxx3 import SolaxX3
+
+from .mysql_data_source import MySQLDataSource
+
+
+def _get_datetime(inverter_time: datetime) -> datetime:
+    return inverter_time - timedelta(hours=TIMEZONE_OFFSET)
+
+
+MYSQL_CONNECTION_INFO = {
+    "user": environ["MYSQL_DB_USERNAME"],
+    "host": environ["MYSQL_DB_HOST_IP"],
+    "password": environ["MYSQL_DB_PASSWORD"],
+}
+DATABASE = environ["MYSQL_DB_DATABASE"]
+TIMEZONE_OFFSET = 2
```

Comment:
consider adding the timezone offset from environment variable, too, but with a default value if the env variable is missing.